automation/test.sh

```diff
@@ -27,6 +27,9 @@ pip3 install --user ansible
 ansible-playbook --version
 ansible-playbook -vvv tests/playbooks/all.yml
 
+# TODO: REMOVE THIS LATER
```

Comment:
This seems to prevent us to get any error. It would be better not to test broken code for devel branch only.

Reply:
Yeah, can we just use github API to fetch issue and skip the test if the issue is open and don't skip if it's closed, that way we now it was fixed properly and we will be testing everything working always